### DISCUSSION TITLE
Já assinou a Carta sobre o Google AMP?

### DISCUSSION THREAD

**Discussion by felipemfp** [frontendbr/forum#890] [Dicas]
> *We are a community of individuals who have a significant interest in the development and health of the World Wide Web (“the Web”), and we are deeply concerned about Accelerated Mobile Pages (“AMP”), a Google project that purportedly seeks to improve the user experience of the Web.*
>
> *In fact, AMP keeps users within Google’s domain and diverts traffic away from other websites for the benefit of Google. At a scale of billions of users, this has the effect of further reinforcing Google’s dominance of the Web.* — leia toda a carta em http://ampletter.org/.

Concordam? 

Outras discussões levantadas por aqui: #175 #854

**Comment by felipechalreo** [comment]
**A Web não é o Google, e não deve ser apenas o Google**. Concordo plenamente, infelizmente muita gente está começando a aceitar/acreditar que a internet se reduz ao Google ou Facebook.

**Comment by didiraja** [comment]
Nunca me incomodou o AMP, muito pelo contrário, mas achei o questionamento completamente pertinente.
